js/jquery.validation.js

```diff
@@ -143,32 +181,38 @@
                 var ajax_method;
                 if ( method === 'POST' || method === 'GET' ) {
                     ajax_method = method;
+
                 } else {
                     // Emulate HTTP method
-                   ajax_method = 'POST';
-                    var found = false;
-                    $.each(data, function(index, item) {
-                        if ( item.name === '_method' ) {
-                            item.value = method;
-                            found = true;
-                            return;
-                        }
-                    });
+                    ajax_method = 'POST';
+
+                    if ( isFormData ) {
+                        data.set('_method',method);
```

Comment:
@PeterChaplin is it possible for isFormData to be true but for data.set() to be not supported ... would this cause prevent PUT requests from working in browsers where this is the case?